Old feature: EXP calculator

My2ndAngelic opened the pull request:
Don't remove it, I found that's very useful for farming.

Yuubari committed:
Merge remote branch 'japanese' from Grabacr07/KanColleViewer

This will definitely fail to build. I'm sure.

Conflicts:
	Grabacr07.KanColleViewer/App.config
	Grabacr07.KanColleViewer/App.xaml.cs
	Grabacr07.KanColleViewer/Models/Settings.cs
	Grabacr07.KanColleViewer/Properties/AssemblyInfo.cs
	Grabacr07.KanColleViewer/SampleData/RepairyardViewModelSampleData.xaml
	Grabacr07.KanColleViewer/ViewModels/Catalogs/SortableColumnViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/Contents/Fleets/FleetViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/Contents/Fleets/ReSortieBarViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/Contents/ToolsViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/MainContentViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/MainWindowViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/SettingsViewModel.cs
	Grabacr07.KanColleViewer/Views/Catalogs/ShipCatalogWindow.xaml
	Grabacr07.KanColleViewer/Views/Catalogs/SlotItemCatalogWindow.xaml
	Grabacr07.KanColleViewer/Views/Contents/Fleets.xaml
	Grabacr07.KanColleViewer/Views/Contents/Homeport.xaml
	Grabacr07.KanColleViewer/Views/Contents/Overview.xaml
	Grabacr07.KanColleViewer/Views/Contents/Quests.xaml
	Grabacr07.KanColleViewer/Views/Controls/KanColleHost.cs
	Grabacr07.KanColleViewer/Views/MainContent.xaml
	Grabacr07.KanColleViewer/Views/MainWindow.xaml
	Grabacr07.KanColleViewer/Views/Settings/Browser.xaml
	Grabacr07.KanColleViewer/Views/Settings/Browser.xaml.cs
	Grabacr07.KanColleViewer/Views/Settings/Operation.xaml
	Grabacr07.KanColleViewer/Views/Settings/Others.xaml
	Grabacr07.KanColleViewer/Views/Settings/Settings.xaml
	Grabacr07.KanColleViewer/Views/StatusBar.xaml
	Grabacr07.KanColleViewer/packages.config
	Grabacr07.KanColleWrapper/KanColleClient.cs
	Grabacr07.KanColleWrapper/Models/Fleet.cs
	Grabacr07.KanColleWrapper/Models/FleetReSortie.cs
	Grabacr07.KanColleWrapper/Models/Raw/kcsapi_createitem.cs
	Grabacr07.KanColleWrapper/Models/Raw/kcsapi_createship.cs
	README.md

Yuubari committed:
Merge: fix some errors, more to come

Yuubari committed:
Merge conflicts resolution, part 2

* Moved CustomSound.cs to WindowsNotifier and adjusted references/packages accordingly

Grabacr07 committed:
建造のネタバレをしていない状態で、完了通知に建造中の艦名が出てしまっていた問題を修正

Yuubari committed:
Re-add options; switch to the new plugin interface for notifications

Yuubari committed:
Restored Language and Updates tabs in Settings, added UI translations

Yuubari committed:
Shipyard: update localisation

Yuubari committed:
Finalise localisation for new strings

Yuubari committed:
Pre-release revision bump

Yuubari committed:
Merge branch 'japanese'

Conflicts:
	Grabacr07.KanColleViewer/Properties/AssemblyInfo.cs

Yuubari committed:
Merge branch 'feature-resourcelogger'

Yuubari committed:
Update build events, fix placement errors in version info

Yuubari committed:
Bump version number in Version.xml

Grabacr07 committed:
リリース用のパッケージ作成スクリプトを用意

Yuubari committed:
Update localisation: morale recovery notification

Yuubari committed:
Allow loading untrusted DLLs

Grabacr07 committed:
プラグインのインターフェイスを整備、通知プラグインの表示画面作成

Yuubari committed:
Restore Flash-related browser settings and refresh on F5

Status bar might also be fixed now.

Yuubari committed:
Fix run-time errors, status bar and status indicators

Yuubari committed:
Fix another error (I need tea)

Grabacr07 committed:
艦娘一覧に AL/MI 出撃マークとフィルター実装

Grabacr07 committed:
環境によって Microsoft.Expresson.Drawing が解決できていないようなので、追加

Grabacr07 committed:
readme.txt の文字コード＆改行コード修正

Grabacr07 committed:
索敵値の計算方法を選択できるように変更

Grabacr07 committed:
索敵計算のミスを修正

Yuubari committed:
Merge branch 'japanese'

Conflicts:
	Grabacr07.KanColleViewer/Properties/AssemblyInfo.cs
	Grabacr07.KanColleViewer/ViewModels/Contents/ToolsViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/SettingsViewModel.cs
	Grabacr07.KanColleViewer/Views/Catalogs/ShipCatalogWindow.xaml

Yuubari committed:
Finalise merge

Yuubari committed:
AL/MI filter localisation

Yuubari committed:
Add new translations (equipment and ships)

Yuubari committed:
Fix autoupdate URL

Grabacr07 committed:
version 3.2 としてリリース

Grabacr07 committed:
連合艦隊関連の API を受け取るためのいくつかのクラスを追加

Yuubari committed:
Merge branch 'japanese' and localise additions

Conflicts:
	Grabacr07.KanColleViewer/Properties/AssemblyInfo.cs
	Grabacr07.KanColleViewer/ViewModels/SettingsViewModel.cs
	Grabacr07.KanColleViewer/Views/Settings/Operation.xaml

Yuubari committed:
Update translations: 2 new quests (one left but probably requires refactoring)

Grabacr07 committed:
航空要員アイコンの追加

Grabacr07 committed:
艦隊の出撃判定方法を変更、連合艦隊の出撃に対応

Grabacr07 committed:
version 3.3 リリース

Yuubari committed:
Merge branch 'japanese'

Conflicts:
	Grabacr07.KanColleViewer/Properties/AssemblyInfo.cs

Yuubari committed:
Bump autoupdate version

Grabacr07 committed:
Overview の艦隊情報での艦名表示で改行されてしまっていた問題の修正

guitarrapc committed:
Bug fix : when targets was copied, empty Directory (not contains any file) will be deleted.

If there are
c:\hoge
c:\hoge\fuga\piyo.exe
Old script will remove c:\hoge as there are no files right under c:\hoge.
Now I added file tree check to prevent these issue

Grabacr07 committed:
Merge pull request #109 from guitarrapc/Bug_fix_for_Package.ps1

Bug fix : Package.ps1 have bug possibility of removing Empty Directory even it's child have file.

Yuubari committed:
Add name trimming in overview

Grabacr07 committed:
Merge remote-tracking branch 'origin/master' into develop

Grabacr07 committed:
パッケージ作成 ps1 の除外条件を修正

twinkfrag committed:
発生中の任務がない場合のために任務確認処理の位置を変更 (fixed #63)

Yuubari committed:
Fix settings to view model mapping, add a default setting for Culture

Yuubari committed:
Add localisation related to LoS display; WIP

Yuubari committed:
Fix margins in the repair/construction overview panels

Yuubari committed:
Fix status bar foreground colour and simplify the definitions

Yuubari committed:
Bump build number

Grabacr07 committed:
Merge pull request #112 from twinkfrag/master

発生中の任務がない場合のために任務確認処理の位置を変更 (fixed #63)

yuyuvn committed:
Add FleetCommandFacility icon

Grabacr07 committed:
Merge pull request #113 from yuyuvn/original-pr

Add FleetCommandFacility icon

Grabacr07 committed:
Merge branch 'master' into develop

Yuubari committed:
Add a new translation

Yuubari committed:
Merge branch 'japanese'

Yuubari committed:
Bump build number for updates from upstream

Grabacr07 committed:
WebBrowser に対する D&D を抑止

fixed #102

Yuubari committed:
Switch to submodules for translations

Yuubari committed:
Switch to submodules for translations, part 2

Yuubari committed:
Remove the event filters

Yuubari committed:
Add orientation setting

This might be changed to allow for more than two layouts later.

Yuubari committed:
Restore the tooltips for the Fleets tab

Yuubari committed:
Add the experience calculator plugin

First version, expect bugs and whatnot.

Yuubari committed:
Bump build number

Yuubari committed:
Update autoupdate URLs to new github URL syntax

Yuubari committed:
Fix URLs in the version file

(Why don't we get rid of these anyway, they're mostly useless as the real autoupdate uses culture?)

Commit:
2014/09/26 アップデートで api_material の内容が読めなくなっていた問題を修正

Commit:
艦娘一覧の出撃海域フィルターを除外

Commit:
version 3.4 リリース

Commit:
readme.txt 更新

Yuubari committed:
Merge branch 'japanese'

Conflicts:
	Grabacr07.KanColleViewer/Properties/AssemblyInfo.cs
	Grabacr07.KanColleViewer/Views/Catalogs/ShipCatalogWindow.xaml

Yuubari committed:
Fix merge error

Yuubari committed:
Remove a duplicate “using” clause

Yuubari committed:
Disable German and Chinese due to no maintainers

Yuubari committed:
Bump build number

Yuubari committed:
Update README.md

Yuubari committed:
Update some translations

Yuubari committed:
Change the wrapper for MSHTML to tlbimp

Wrapper “primary” fails in Visual Studio 2013.3 on Windows 10 Technical
Preview build 9860.

All of mshtml → MSHTML case changes in the affected .cs files are due to
above change.

Yuubari committed:
Add a toggle for stats column in equipment list (WIP)

Yuubari committed:
Add stats visibility toggle for equipment list: finalised

Yuubari committed:
Update project files to produce a clean output

Yuubari committed:
Add status indicator for critical damage cases

Yuubari committed:
Update API interface: item level and item lock

Yuubari committed:
Add on-the-fly updates for HQ experience, level and rank

Yuubari committed:
API change: ship no longer has sokuh. Now calculating using soku.

Presently assuming anything above 5 to be fast.

Yuubari committed:
Use quest IDs when looking up translations

Yuubari committed:
Update translations, bump translations' versions

Yuubari committed:
Update README

kbinani committed:
Take the total amount of exp. account into level sort

Yuubari committed:
Bump quests and ship names versions

Yuubari committed:
Bump quest translations version

Yuubari committed:
Bump quests version, fixing a mistake

Yuubari committed:
Update translations

Yuubari committed:
Add AAFD icon by yuyuvn

Yuubari committed:
Bump KCV version + quest translations version

Yuubari committed:
Bump quest translations version

Yuubari committed:
Bump quests translations version

Yuubari committed:
Bump quests version. Quest IDs are finalised.

Yuubari committed:
Fix issues with text trimming, take 1

Yuubari committed:
Add a FAQ

Yuubari committed:
Fix a formatting error

Yuubari committed:
Fix ampersand

Yuubari committed:
Update README.md

Yuubari committed:
Fix issues with text trimming, take 2

Yuubari committed:
Bump quest translations version

Yuubari committed:
Bump version due to a missing tag

Yuubari committed:
Bump ship translations version, change versions URL

Yuubari committed:
API change: api_defeq isn't present in api_mst_ship

Yuubari committed:
Fix tabulation

Forgot to fix tabulation settings after reinstalling VS2013.

Yuubari committed:
Update models for API changes

Yuubari committed:
Bump translations' versions

Yuubari committed:
Update FAQ.md

Yuubari committed:
Update FAQ.md

Yuubari committed:
Bump quests and ships versions

Yuubari committed:
Merge branch 'master' of https://github.com/Yuubari/KanColleViewer

Yuubari committed:
Bump quests and equipment versions

Yuubari committed:
Update FAQ.md

Yuubari committed:
Update FAQ.md

Yuubari committed:
Update README.md

Yuubari committed:
Update README.md

KyoriAsh committed:
Translate leftover

Yuubari committed:
Merge pull request #37 from KyoriAsh/patch-1

Translate leftover

Grabacr07 committed:
C# 6.0

Grabacr07 committed:
装備アイコン追加

Grabacr07 committed:
表示の微調整、version 3.5 リリース

Grabacr07 committed:
defeq がなくなってる件、とりあえず雑対応 (雑)

Yuubari committed:
Bump translations versions

Yuubari committed:
Merge remote-tracking branch 'upstream/master' into grabacr-3.5

Conflicts:
	Grabacr07.KanColleViewer/Models/ProductInfo.cs
	Grabacr07.KanColleViewer/Themes/Generic.SlotItemIcon.xaml
	Grabacr07.KanColleViewer/ViewModels/Catalogs/ShipCatalogSortWorker.cs
	Grabacr07.KanColleViewer/ViewModels/Catalogs/SlotItemViewModel.cs
	Grabacr07.KanColleViewer/Views/Contents/Overview.xaml
	Grabacr07.KanColleWrapper/Models/Admiral.cs
	Grabacr07.KanColleWrapper/Models/Quest.cs
	Grabacr07.KanColleWrapper/Models/Ship.cs
	Grabacr07.KanColleWrapper/Models/ShipInfo.cs
	Grabacr07.KanColleWrapper/Models/ShipType.cs
	Grabacr07.KanColleWrapper/Models/SlotItem.cs
	Grabacr07.KanColleWrapper/Models/SlotItemInfo.cs
	Grabacr07.KanColleWrapper/Models/SlotItemType.cs

Yuubari committed:
Add gitignore for .sln.ide of VS2015

Yuubari committed:
Bump version, update translations submodule

Grabacr07 committed:
速力判定の修正

fixed #122

Yuubari committed:
Merge remote-tracking branch 'upstream/master' into grabacr-3.5

Conflicts:
	Grabacr07.KanColleWrapper/Models/ShipInfo.cs

Yuubari committed:
Merge branch 'grabacr-3.5'

Grabacr07 committed:
TargetPlatformVersion を削除

Grabacr07 committed:
艦隊タブを新しくしてみるテスト

Yuubari committed:
Merge remote-tracking branch 'upstream/master'

Yuubari committed:
Restore zh-CN, fix lost tooltip after merge

Yuubari committed:
Add support for negative stats

Grabacr07 committed:
連合艦隊対応ちゅう

Grabacr07 committed:
艦隊タブ再考

Grabacr07 committed:
Merge branch 'develop2' into develop

Grabacr07 committed:
Error "csc2.exe" exited with code 1. がひどすぎるので諦め

Grabacr07 committed:
C# 5.0

Yuubari committed:
Add SampleData to Overview

Yuubari committed:
Update FAQ.md

Grabacr07 committed:
索敵値計算ロジックに [2-5 式 (秋)] を追加

Yuubari committed:
Merge branch 'master' of https://github.com/Yuubari/KanColleViewer

Grabacr07 committed:
MMCSS

Grabacr07 committed:
索敵選択ロジックの修正 (UI がまだ)

Grabacr07 committed:
索敵計算 2-5 秋対応

Grabacr07 committed:
改修工廠で改修に使用した装備が装備数に反映されない問題の修正

#121

Grabacr07 committed:
母港に戻ったタイミングで提督経験値等を反映するように修正

Grabacr07 committed:
version 3.6

Grabacr07 committed:
api_use_slot_id の null チェック

Grabacr07 committed:
護衛退避検出実装 (動作未確認)

Grabacr07 committed:
文字列でプロパティ名指定するのもうやだ (´･ω･`)

Grabacr07 committed:
fleetwindow

Grabacr07 committed:
艦娘一覧の装備表示修正

fixed #125

Yuubari committed:
Merge remote-tracking branch 'upstream/master' into grabacr-3.6

Conflicts:
	Grabacr07.KanColleViewer/Models/ProductInfo.cs
	Grabacr07.KanColleViewer/Properties/AssemblyInfo.cs
	Grabacr07.KanColleViewer/ViewModels/Catalogs/ShipCatalogSortWorker.cs
	Grabacr07.KanColleViewer/ViewModels/Catalogs/SlotItemViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/SettingsViewModel.cs
	Grabacr07.KanColleViewer/Views/Catalogs/ShipCatalogWindow.xaml
	Grabacr07.KanColleViewer/Views/Settings/Operation.xaml
	Grabacr07.KanColleWrapper/Homeport.cs
	Grabacr07.KanColleWrapper/KanColleClient.cs
	Grabacr07.KanColleWrapper/Models/Admiral.cs
	Grabacr07.KanColleWrapper/Models/Quest.cs
	Grabacr07.KanColleWrapper/Models/Ship.cs
	Grabacr07.KanColleWrapper/Models/ShipInfo.cs
	Grabacr07.KanColleWrapper/Models/ShipType.cs
	Grabacr07.KanColleWrapper/Models/SlotItem.cs
	Grabacr07.KanColleWrapper/Models/SlotItemInfo.cs

Grabacr07 committed:
連合艦隊対応のため Fleet 型周辺の内部実装を大幅に変更

Grabacr07 committed:
不具合修正

Grabacr07 committed:
更に不具合修正

Grabacr07 committed:
Index を ID にするの忘れてた (そして相変わらず動作未確認)

Grabacr07 committed:
艦隊詳細ウィンドウ作りかけ

Yuubari committed:
Update FAQ.md

Clarify the stance on .exe.config modification.

Grabacr07 committed:
艦隊詳細、艦隊選択

Grabacr07 committed:
艦隊詳細ウィンドウに状態インジケーター追加

Grabacr07 committed:
艦隊詳細ウィンドウにいろいろ追加

Grabacr07 committed:
CombinedFleet の FleetState が正しく Update() と Calc() されるように修正

Grabacr07 committed:
ハイパー作りかけマン

Grabacr07 committed:
艦隊詳細ウィンドウとりあえず版

Grabacr07 committed:
コミット漏れ

Grabacr07 committed:
デザイン時データ消し忘れマン

Grabacr07 committed:
色付けたりしてみた

Grabacr07 committed:
連合艦隊の艦隊名変更通知できていなかった問題

Grabacr07 committed:
不具合修正

Grabacr07 committed:
めも

Grabacr07 committed:
艦隊詳細ウィンドウでタッチ スクロールのサポート

Grabacr07 committed:
思いついたとこだけ直した

Grabacr07 committed:
Merge pull request #118 from kbinani/sort_by_exp

Take the total amount of exp. account into level sort

Grabacr07 committed:
Merge branch 'master' into develop

Grabacr07 committed:
ソート機能更新

Grabacr07 committed:
level を受け入れられるようにした

Yuubari committed:
Merge remote-tracking branch 'upstream/master'

Conflicts:
	Grabacr07.KanColleViewer/ViewModels/Catalogs/SortableColumnViewModel.cs

Grabacr07 committed:
列ヘッダーからの艦種ソートをゲーム内の動作と一致するように修正

 #129

Grabacr07 committed:
既存の装備名表示にレベル表示を追加

Grabacr07 committed:
add .editorconfig

Grabacr07 committed:
Ship / Fleet の Situation 表示の修正

Grabacr07 committed:
Merge branch 'develop' into feature/itemlevel

Grabacr07 committed:
装備一覧画面で改修表示対応

Grabacr07 committed:
出撃可能判定にコンディション入れ忘れマン

#126

Grabacr07 committed:
初期状態でレベル降順ソートに

Grabacr07 committed:
Merge branch 'feature/shipsort' into develop

Grabacr07 committed:
version 3.7 beta

veigr committed:
マスタに海域、マップデータを追加

Yuubari committed:
Update FAQ.md

Grabacr07 committed:
Merge pull request #130 from veigr/add-map-master

マスタに海域、マップデータを追加

Grabacr07 committed:
Merge branch 'master' into develop

Yuubari commented:
This repository is dead since Zharay quit. It's rather pointless to try to send PRs to him.

Yuubari committed:
Reimplement shortcut stats for natural ASW, LoS and Evasion of shipgirls

Refactored using iterators over the new ShipSlot-based EquippedSlots array.

Yuubari committed:
Merge remote-tracking branch 'upstream/master'

Yuubari committed:
Add a check for Flash® ActiveX, bump assembly version

KCV will now check if Adobe® Flash® Player ActiveX element is installed; if it isn't, the user will be presented with a message box prompting them to download it. On clicking “Yes”, KCV will close and spawn a Microsoft® Internet Explorer® window with the Adobe® Flash® download page.

Yuubari committed:
水上艦要員のアイコンを追加

Grabacr07 committed:
Merge pull request #133 from Yuubari/grabacr

水上艦要員のアイコンを追加

Grabacr07 committed:
R# に #if を消されてた件

Grabacr07 committed:
Merge branch 'master' into develop

Conflicts:
	Grabacr07.KanColleWrapper/Models/SlotItemIconType.cs

Yuubari committed:
Make Fiddler only listen on localhost

Grabacr07 committed:
艦隊詳細のレイアウト変更

Grabacr07 committed:
装備改修対応

Grabacr07 committed:
艦隊詳細レイアウト更新

Grabacr07 committed:
艦隊詳細、縦に優しくしてみた

Grabacr07 committed:
readme.txt 更新

Yuubari committed:
Merge branch 'upstream/master'

Conflicts:
	Grabacr07.KanColleViewer/Models/ProductInfo.cs
	Grabacr07.KanColleViewer/Properties/AssemblyInfo.cs
	Grabacr07.KanColleViewer/ViewModels/Catalogs/ShipCatalogSortTarget.cs
	Grabacr07.KanColleViewer/ViewModels/Catalogs/ShipCatalogSortWorker.cs
	Grabacr07.KanColleViewer/ViewModels/Catalogs/ShipCatalogWindowViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/Catalogs/SlotItemCatalogViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/Catalogs/SlotItemViewModel.cs
	Grabacr07.KanColleViewer/ViewModels/Catalogs/SortableColumnViewModel.cs
	Grabacr07.KanColleViewer/Views/Catalogs/ShipCatalogWindow.xaml
	Grabacr07.KanColleViewer/Views/Catalogs/SlotItemCatalogWindow.xaml
	Grabacr07.KanColleWrapper/Itemyard.cs
	Grabacr07.KanColleWrapper/Models/Raw/kcsapi_slotitem.cs
	Grabacr07.KanColleWrapper/Models/Ship.cs
	Grabacr07.KanColleWrapper/Models/SlotItem.cs
	Grabacr07.KanColleWrapper/Models/SlotItemInfo.cs

Yuubari committed:
Fix merge issues, clean up code from some remnants

Some features are unavailable now, to be fixed later.

Yuubari committed:
Remove Experience Calculator for now; clean up

Yuubari committed:
Fix status line, localise fleet details

Yuubari committed:
大型飛行艇のアイコンを追加

Yuubari committed:
Merge branch 'grabacr'

Yuubari committed:
Localise fleet details button

Yuubari committed:
Fix an oversight in ship status localisation

Grabacr07 committed:
Merge pull request #144 from Yuubari/icon-flyingboat

大型飛行艇のアイコンを追加

Grabacr07 committed:
5/18 アップデート対応

Grabacr07 committed:
version 3.8

Yuubari committed:
Merge remote-tracking branch 'remotes/upstream/master'

Yuubari committed:
Update README

Yuubari committed:
Bump assembly version, fix sorting, fix flicking

Grabacr07 committed:
fixed #145

Grabacr07 committed:
出撃中に艦隊ステータスが更新されない問題の修正

Grabacr07 committed:
version 3.8.1

Yuubari committed:
Use a different settings path

Settings folder path changed to avoid version conflicts with Grabacr07's original builds and other forks.

Yuubari committed:
Update FAQ.md

Yuubari committed:
Update FAQ.md

Forgot to edit the section header.

Yuubari committed:
Merge remote-tracking branch 'remotes/upstream/master'

Conflicts:
	Grabacr07.KanColleViewer/Properties/AssemblyInfo.cs

Yuubari committed:
Attempt to re-implement item stats display

Yuubari committed:
Re-implement visibility toggling for columns

Yuubari committed:
Ship roster: fix localisation; restore evasion+ASW

Yuubari committed:
Equipment window title localisation

Yuubari committed:
Fixed repair time calculation for CAV, AS, AR.

Yuubari committed:
Add remodel level to overview stats tooltips

The level is only displayed if it's not the final remodel for that particular shipgirl.

Yuubari committed:
Fix: whitespace used instead of tabulation

philippeop committed:
Translate notification tab in startup settings page

philippeop committed:
Translated notification pane text

philippeop committed:
Tab usage instead of whitespace

Yuubari committed:
Merge pull request #70 from tasetase/fix_trans

Couple of missing translations fixes

philippeop committed:
Fixed the quick status of the fleet almost always showing "Morale recovered" instead of the actual state.

Yuubari committed:
Update the start-up URL

Yuubari committed:
Merge pull request #72 from tasetase/fleet-status-fix

Fixed the quick status of the fleet

Yuubari committed:
Update README.md

Yuubari committed:
Update README.md
